python/mxnet/numpy/multiarray.py

```diff
@@ -5866,7 +6305,7 @@ def trace(a, offset=0, axis1=0, axis2=1, out=None):
 
 
 @set_module('mxnet.numpy')
-def transpose(a, axes=None):
+def transpose(a: array, axes=None):
```

Comment:
Currently, we do not need to add type hint as some of the CI is still using Python < 3.8